pkg/nodes/nodes.go

```diff
@@ -585,19 +459,9 @@ func (n *NodeManager) IncreaseBytes(node OsqueryNode, incBytes int) error {
 	return nil
 }
 
-// ConfigRefresh to perform all needed update operations per node in a config request
-func (n *NodeManager) ConfigRefresh(node OsqueryNode, lastIp string, incBytes int) error {
-	updates := map[string]interface{}{
-		"last_config":    time.Now(),
-		"bytes_received": node.BytesReceived + incBytes,
-	}
-	if lastIp != "" {
-		updates["ip_address"] = lastIp
-	}
-	if err := n.DB.Model(&node).Updates(updates).Error; err != nil {
-		return fmt.Errorf("Updates %v", err)
-	}
-	return nil
+func (n *NodeManager) RefreshLastSeenBatch(nodeID []uint) error {
+
+	return n.DB.Model(&OsqueryNode{}).Where("id IN ?", nodeID).UpdateColumn("last_config", time.Now()).Error
```

Comment:
I think we can just have a new field called `last_seen` and not to recycle any of the existing ones.